README.md

```diff
@@ -1,146 +1,129 @@
+We have used this to 
 
-## Overview
+ - Enable automated functional black and grey box testing of components. 
+ - Enable automated visual regression of components with [Dextrose]
 
-Fructose has 2 main parts to it.
-  - The app
-  - The test api
+# Overview
 
-You need to use both parts to create and run a fructose test.
+## The App
 
-The app contains all the components under test and a mechanism to load them from a client.
+The Fructose app allows for the loading of arbitrary components at runtime. Once the app is started you are able to load the components in either manual or automated fashion.
 
-The test api enables you to easily load components into the app from a test.
+<img src="https://imgur.com/VJR5Tbz.gif">
+
+Swipe right to manually load a component (shown above)
 
-To interact with the component under test you must use a tool such as [detox], [chromeless] or [appium].
 
 ## Getting Started
 
-### Install
+We are actively working on this to become easier. 
 
-```
-yarn add fructose --dev
-yarn add react-native-storybook-loader --dev
-```
 
-### Set up the app
+## Running and understanding the examples
 
-Create a folder `.fructose` in your project root directory.
+The best way to understand how fructose works is to run the end to end tests.
 
-Create a `index.*.js` in this folder for your platform of choice: `ios`, `android`, or `web`.
 
-These files will load the components to be tested into the Fructose app.
+### Running
+From the root of the project run `yarn` and then either...
 
-Register the component to the name that your app binary expects.
+`yarn e2e:test:web`
 
-```
-import { AppRegistry } from "react-native";
-import Fructose from "fructose";
-import { loadStories } from './components';
+`yarn e2e:test:ios`
 
-AppRegistry.registerComponent("e2eTests", () => Fructose(loadStories));
-```
+`yarn e2e:test:android`
 
-We recommend looking at the examples for [web](./e2eTests/.fructose/index.web.js) and [native](./e2eTests/.fructose/index.ios.js) as there are slight differences.
 
-### Set up the tests
+You will need a (simulator | emulator | chrome) to run these examples.
+### Understanding
 
-Add a setup file in `.fructose`. This is a file that should run once before any of your tests run.
+In the `e2e Test` folder you will find examples of how to consume fructose.
+We suggest first looking at the `scripts` folder to understand the steps required for running tests with Fructose.
 
-In the below examples we will provide example config for jest. If you use a different test runner then you can do the equivalent for that runner.
+Then look at the `examples` folder. 
 
-In this file you will need to run test hooks.
+The `.showcase` file is a description of how a component should be rendered.
+`showcase` files came around because originally fructose consumed `storybook` files. This eventaully resulted in numerous issues so we created an abstraction layer which decouples components from storybook.
```

Comment:
eventually

Reply:
If we're giving @L0wry English lessons, we might as well go the distance and get him to remove the word 'eventually' altogether. I feel like you guys would enjoy the elements of style. The granddaddy of style guides. https://www.goodreads.com/book/show/33514.The_Elements_of_Style 

Yes, I'm enjoying doing this PR way too much. It's lonely in Dubai.